backend/src/main/java/com/itlize/joolemarketplace/service/impl/ProjectServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.itlize.joolemarketplace.service.impl;
+
+import com.itlize.joolemarketplace.model.Project;
+import com.itlize.joolemarketplace.repository.ProjectRepository;
+import com.itlize.joolemarketplace.repository.UserRepository;
+import com.itlize.joolemarketplace.service.ProjectService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ProjectServiceImpl implements ProjectService {
+    @Autowired
+    private ProjectRepository projectRepository;
+    @Autowired
+    private UserRepository userRepository;
```

Comment:
You probably forgot to delete this? It looks like this class doesn't use `userRepository`